src/main/java/pl/bajerska/befindyourmeal/edamam/ApiConnection.java

```diff
@@ -0,0 +1,6 @@
+//package pl.bajerska.befindyourmeal.edamam;
```

Comment:
Not needed files can be removed from main branch. But can be keeped in branches (where needed) or stashed (`git stash`).